test/runtests.jl

```diff
@@ -257,6 +259,54 @@ mkpath(TEST_EXAMPLES_DIR)
     end
 
   end
+
+  # Test set for PolyData
+  @testset "PolyData" begin
+    ## Generate sample vtp file
+    
+    # define points
+    n = 20
+    points = zeros(3,n)
+    points[1,:] .= [cos(4*pi*i/n) for i in 0:n-1]
+    points[2,:] .= [sin(4*pi*i/n) for i in 0:n-1]
+    points[3,:] .= [0.2i for i in 0:n-1]
```

Comment:
Some minor formatting stuff that can help quite a bit when working in larger projects (also below). 
```suggestion
    points = zeros(3, n)
    points[1, :] .= [cos(4 * pi * i / n) for i in 0:(n - 1)]
    points[2, :] .= [sin(4 * pi * i / n) for i in 0:(n - 1)]
    points[3, :] .= [0.2 * i for i in 0:n-1]
```